tutorial/clock.h

```diff
@@ -2,14 +2,16 @@
 // milliseconds.
 
 #ifdef _WIN32
-extern "C" bool QueryPerformanceCounter(uint64_t *);
-extern "C" bool QueryPerformanceFrequency(uint64_t *);
+#include <Windows.h>
```

Comment:
Because the current version in master does not compile. I do not remember the exact error but the function prototypes defined originally did not match what was being used elsewhere and it caused a linking error.